### PR TITLE
refactor(app): update connect via usb screen

### PR DESCRIPTION
**Pull request by koji**
# Overview
update connect via usb screen to align with the hi-fi design.
This PR is only update the UI part. The functionality of USB connection part will be implemented later.

FYI
For this update, I updated `connected screen` and `not connected screen` separately. Actually we can use optional to merge both screens but I think this way can keep readability. And the component itself around 100 lines.
 

# Test Plan
1. Network setup menu
2. Tap `USB`

To see the success case this pr needs to change the line.
line27  `const isConnected = false` => true

# Changelog
- update ConnectViaUSB component and test

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] update the text for unconnected case

**Comment by ewagoner** [comment]
Screenshot 2023-05-18 at 4 19 12 PM

The success screen should have centered text.

**Comment by koji** [comment]
> Screenshot 2023-05-18 at 4 19 12 PM
> 
> The success screen should have centered text.

@ewagoner  thanks.
just fixed

File: app/src/pages/OnDeviceDisplay/ConnectViaUSB.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export function ConnectViaUSB(): JSX.Element {
   const { i18n, t } = useTranslation(['device_settings', 'shared'])
   const history = useHistory()
   // ToDo (kj:05/16/2023) isConnected part will be implemented later
-  const isConnected = false
+  const isConnected = true
```
Comment (ewagoner) [inline app/src/pages/OnDeviceDisplay/ConnectViaUSB.tsx]: Oops :)

Comment (koji) [reply app/src/pages/OnDeviceDisplay/ConnectViaUSB.tsx]: just switched back to false.
Brent opened the USB connection detect function for the Desktop so probably this component will have the functionality soon.

**Review by ewagoner — APPROVED** [review]

**Pull request merged**